src/main/java/uk/gov/hmcts/darts/common/repository/EventRepository.java

```diff
@@ -53,6 +53,7 @@ SELECT EXISTS (
          SELECT new uk.gov.hmcts.darts.event.model.EventSearchResult(
             e.id,
             e.createdDateTime,
```

Comment:
You've removed ChronicleID and Antecedent ID from the openapi code further down, so wondered why that's been removed here as well? Or will that be in a future PR?

Reply:
I have updated the code to also remove this